### DISCUSSION TITLE
What are the possible tags in the fdm_config version/release attribute and their significance?

### DISCUSSION THREAD

**Discussion by Octal450** [JSBSim-Team/jsbsim#1168] [Support requests]
Hi,
In release:
I've seen ALPHA BETA and PRODUCTION, but I can't find any documentation on this.

In version:
I've only seen 2.0.

Are these just notes, or do they affect something in JSBSim? I have been using "BETA" and 2.0.

<fdm_config name="MD-11F-GE" version="2.0" release="BETA">

Many thanks,
Josh

**Comment by seanmcleod** [comment]
The only use of `release` is to print the contents out to the console when processing the aircraft's FDM.

https://github.com/search?q=repo%3AJSBSim-Team%2Fjsbsim+release+language%3AC%2B%2B&type=code&l=C%2B%2B

Gets read in here:

https://github.com/JSBSim-Team/jsbsim/blob/57361d0756a7a4d477d80f786e381317d8722671/src/FGFDMExec.cpp#L1202

And gets output via the logging system here, with some varying colour depending on the value, and some warning text to the user.

https://github.com/JSBSim-Team/jsbsim/blob/57361d0756a7a4d477d80f786e381317d8722671/src/FGFDMExec.cpp#L1218-L1249

**Comment by Octal450** [comment]
Thanks for the explanation.

J

**Comment by Octal450** [comment]
Oh, what about version?

From my quick look through, doesn't look to be significant just a print out, but want to double check.

J
